src/renderer/utils/refreshToken.js

```diff
@@ -2,27 +2,32 @@ import axios from "axios";
 
 const refreshToken = async () => {
   try {
-    const userRefreshToken = window.localStorage.getItem("refreshToken");
-    const userId = window.localStorage.getItem("userId");
-    const authorization = "Bearer " + userRefreshToken;
-    const { jwtToken, refreshToken } = await axios.post(
-      `${import.meta.env.VITE_SERVER_URL}/auth/refresh/kakao`,
-      { userId },
-      {
-        headers: {
-          "Content-Type": "application/json",
-          authorization,
-        },
-      },
+    const deliOrderUserId = window.localStorage.getItem("deliOrderUserId");
+    const deliOrderRefreshToken = window.localStorage.getItem(
+      "deliOrderRefreshToken",
     );
-
-    if (refreshToken) {
-      localStorage.setItem("refreshToken", refreshToken);
+    const authorization = "Bearer " + deliOrderRefreshToken;
+    if (deliOrderUserId && deliOrderRefreshToken) {
+      const { newDeliOrderToken, newDeliOrderRefreshToken } = await axios.post(
+        `${import.meta.env.VITE_SERVER_URL}/auth/token/refresh`,
+        { deliOrderRefreshToken, deliOrderUserId },
+        {
+          headers: {
+            "Content-Type": "application/json",
+            authorization,
+          },
+        },
+      );
+      window.localStorage.setItem("deliOrderToken", newDeliOrderToken);
+      window.localStorage.setItem(
+        "deliOrderRefreshToken",
+        newDeliOrderRefreshToken,
+      );
```

Comment:
```suggestion
    const authorization = "Bearer " + deliOrderRefreshToken;
    
    if (deliOrderUserId && deliOrderRefreshToken) {
      const { newDeliOrderToken, newDeliOrderRefreshToken } = await axios.post(
        `${import.meta.env.VITE_SERVER_URL}/auth/token/refresh`,
        { deliOrderRefreshToken, deliOrderUserId },
        {
          headers: {
            "Content-Type": "application/json",
            authorization,
          },
        },
      );
      
      window.localStorage.setItem("deliOrderToken", newDeliOrderToken);
      window.localStorage.setItem(
        "deliOrderRefreshToken",
        newDeliOrderRefreshToken,
      );
```
소소하지만 여기서는 조건문 앞뒤로 개행 해주시면 가독성에 도움이 될 것 같습니다 🤭.

Reply:
제안해주신대로 개행을 추가해서 가독성을 개선해보도록 하겠습니다😃